java-rx-smgrsdk/src/main/java/io/vantiq/util/VertxWideData.java

```diff
@@ -0,0 +1,140 @@
+package io.vantiq.util;
+
+import com.google.common.base.Ticker;
+import com.google.common.cache.Cache;
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.RemovalListener;
+import com.google.common.cache.Weigher;
+import io.vertx.core.Vertx;
+import io.vertx.core.shareddata.LocalMap;
+import io.vertx.core.shareddata.Shareable;
+
+import java.util.Collection;
+import java.util.Objects;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.function.Supplier;
+
+/**
+ * Atomic, shareable object reference.<br/>
```

Comment:
wondering of the comment content shouldn't be aligned with the other ones in the repo (copyright / dates)